app/scripts/services/rest.js

```diff
@@ -36,6 +36,22 @@ export function Api ($rootScope, $resource, config) {
 
     Transactions: $resource(server + '/transactions/:transactionId', { transactionId: '@_id' }),
 
+    TransactionBodies: function(transactionId, bodyId, start, end) {
+      const headers = {}
+      if (
+        end &&
```

Comment:
I think this should be `start && end && end > start` right?

```suggestion
        start &&
```